src/picongpu/include/plugins/IsaacPlugin.hpp

```diff
@@ -400,12 +403,10 @@ class IsaacPlugin : public ILightweightPlugin
                 json_object_set_new( visualization->getJsonMetaRoot(), "cell count", json_string( "Total numbers of cells" ) );
                 json_object_set_new( visualization->getJsonMetaRoot(), "particle count", json_string( "Total numbers of particles" ) );
             }
-            if (visualization->init() != 0)
+            if (visualization->init(reconnect?RetryEverySend:ReturnAtError) != 0)
```

Comment:
Actually I did, and `constexpr` functions will work with both:
  https://youtu.be/EkdfiHs78DY?t=10m17s